source/mir/random/flex/package.d

```diff
@@ -110,6 +111,10 @@ Returns:
 auto flex(F0, F1, F2, S)
                (in F0 f0, in F1 f1, in F2 f2,
                 S c, S[] points, S rho = 1.1)
+    if (isFloatingPoint!(ReturnType!F0) &&
```

Comment:
Just remove this constraints. They would not work for templated opCall for example